regression/contracts/invar_havoc_dynamic_multi-dim_array_all_const_idx/test.desc

```diff
@@ -1,12 +1,14 @@
-KNOWNBUG
+CORE
```

Comment:
Should be merged with the commit that actually fixes the issue to enable this test.

Reply:
Done; thanks!